workshops/templates/base.html

```diff
@@ -64,17 +68,29 @@
           </form>
 
           <ul class="nav navbar-nav navbar-right">
+            {% if perms.workshops.add_event or perms.workshops.add_host or perms.workshops.add_person or perms.workshops.add_airport %}
```

Comment:
Again, show them but gray them out. Also, is there an easy way to combine multiple atomic permissions into a single permission so that you can say "if perms.workshops.add_anything" ?  Because spelling out the entire range of atomic permissions in a template strikes me as error-prone.